src/wallet/feebumper.cpp

```diff
@@ -30,7 +30,7 @@ int64_t CalculateMaximumSignedTxSize(const CTransaction &tx, const CWallet *pWal
     for (auto& input : tx.vin) {
         const auto mi = pWallet->mapWallet.find(input.prevout.hash);
         assert(mi != pWallet->mapWallet.end() && input.prevout.n < mi->second.tx->vout.size());
-        vCoins.emplace_back(std::make_pair(&(mi->second), input.prevout.n));
+        vCoins.emplace_back(&(mi->second), input.prevout.n);
```

Comment:
In commit "Use static calls for GetRequiredFee and GetMinimumFee, remove make_pair from emplace_back"

Could drop parens around mi->second. (Just an observation, feel free to leave.)